storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -398,6 +398,7 @@ func (s *fileSystemSource) loadShardReadersDataIntoShardResult(
 		var (
 			timeRange = r.Range()
 			start     = timeRange.Start
+			blockSize = timeRange.End.Sub(start)
```

Comment:
Can we use the namespace options here too, i.e. `ns.Options().RetentionOptions().BlockSize()`?